File: Computer-Seekho-Java/ComputerSeekho/src/main/java/com/Project/DTO/StudentDto.java
package com.Project.DTO;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class StudentDto {
    private int id;
    private String photoUrl;
    private String name; 
    private String mobile;
    private String course;
    private String batch;
    private int pendingFees;
}

File: Computer-Seekho-.NET/Computer_Seekho_DN/DTO/StudentDto.cs
namespace Computer_Seekho_DN.DTO;

public class StudentDto
{
    public int Id { get; set; }
    public string PhotoUrl { get; set; }
    public string Name { get; set; }
    public string Mobile { get; set; }
    public string Course { get; set; }
    public string Batch { get; set; }
    public int PendingFees { get; set; }

    // Default Constructor
    public StudentDto() { }

    // Parameterized Constructor
    public StudentDto(int id, string photoUrl, string name, string mobile, string course, string batch, int pendingFees)
    {
        Id = id;
        PhotoUrl = photoUrl;
        Name = name;
        Mobile = mobile;
        Course = course;
        Batch = batch;
        PendingFees = pendingFees;
    }
}

